ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino.Receive.cs

```diff
@@ -607,8 +610,7 @@ void FlattenConvertedObject(object item)
           // handle render material
           if (render != null)
           {
-            var convertedMaterial = converter.ConvertToNative(render) as RenderMaterial; //Maybe wrap in try catch in case no conversion exists?
-            if (convertedMaterial != null)
+            if (converter.ConvertToNative(render) is RenderMaterial convertedMaterial)
```

Comment:
@AlanRynne do you mean that if render is not RenderMaterial this an error as we're expecting it to actually BE one?

Reply:
> since `ConvertToNative` is quite a critical operation, i feel like this is hiding it a bit. I like the change for this to be an if statement, but maybe we should do:
> 
> ```cs
> var result = converter.ConvertToNative(render);
> if(render is RenderMaterial converterMaterial){...}
> ```
> 
> I don't have very strong feelings about this though.

happy to make this change, I also don't have strong feelings about it